extensions/amp-story/1.0/amp-story-reaction.js

```diff
@@ -56,9 +61,7 @@ export let ReactionOptionType;
 
 /**
  * @typedef {{
- *    totalResponseCount: number,
- *    hasUserResponded: boolean,
- *    responses: !Array<ReactionOptionType>,
+ *    options: !Array<ReactionOptionType>
```

Comment:
What's motivating this change? Why would we remove the `hasUserResponded` and `totalResponseCount`? Processing these server side sounds like a good thing.

Reply:
We talked about the backend using always 4 options for the quiz schema. If an option gets removed in the story frontend, the backend would return many votes accounted for on the totalResponseCount that wouldn't appear so the percentages could not add up. This can be solved by only adding up the counts of the options that do appear on the frontend, which is why it wouldn't be needed to be processed server side.